### PR TITLE
Cross compile with the new collections

### PR DESCRIPTION
**Pull request by julienrf**
Do not merge until 2.13 is released.

### CODE REVIEW

File: shared/src/main/scala-2.11/scala/util/parsing/input/PagedSeq.scala [hunk 1]
```diff
@@ -0,0 +1 @@
+../../../../../scala-2.12/scala/util/parsing/input/PagedSeq.scala
```
Comment (julienrf) [inline shared/src/main/scala-2.11/scala/util/parsing/input/PagedSeq.scala]: I moved `PagedSeq.scala` to the `src/main/scala-2.12` source directory and symlinked it from here (`scala-2.11` source directory)

File: shared/src/main/scala-2.13.0-pre-47ffa9c-SNAPSHOT/scala/util/parsing/input/PagedSeq.scala [hunk 2]
```diff
@@ -0,0 +1,274 @@
...
+   with scala.collection.IndexedSeq[T]
+{
+  def this(more: (Array[T], Int, Int) => Int) = this(more, new Page[T](0), 0, UndeterminedEnd)
+
+  // Members declared in scala.collection.Seq
+  def iterableFactory: collection.SeqFactory[collection.IndexedSeq] = collection.IndexedSeq
+
+  // Members declared in scala.collection.IterableOps
+  protected[this] def fromSpecificIterable(coll: collection.Iterable[T]): collection.IndexedSeq[T] = iterableFactory.from(coll)
+  protected[this] def newSpecificBuilder(): collection.mutable.Builder[T, collection.IndexedSeq[T]] = iterableFactory.newBuilder()
```
Comment (julienrf) [inline shared/src/main/scala-2.13.0-pre-47ffa9c-SNAPSHOT/scala/util/parsing/input/PagedSeq.scala]: The only difference with the 2.11 and 2.12 implementation is these 3 lines above that have been added.

File: shared/src/main/scala-2.11/scala/util/parsing/input/PagedSeq.scala (hunk 1 above)
Comment (lrytz) [reply shared/src/main/scala-2.11/scala/util/parsing/input/PagedSeq.scala]: This probably doesn't work on Windows. Alternatively we can do something like https://github.com/sbt/sbt/issues/2310

File: shared/src/main/scala-2.13.0-pre-47ffa9c-SNAPSHOT/scala/util/parsing/input/PagedSeq.scala (hunk 2 above)
Comment (lrytz) [reply shared/src/main/scala-2.13.0-pre-47ffa9c-SNAPSHOT/scala/util/parsing/input/PagedSeq.scala]: What do you think about moving those methods to a separate trait in a separate source file? In versions up to 2.12 the source file would be empty. Then we can avoid duplicating the entire file.

Comment (julienrf) [reply shared/src/main/scala-2.13.0-pre-47ffa9c-SNAPSHOT/scala/util/parsing/input/PagedSeq.scala]: Done!

File: shared/src/main/scala-2.11/scala/util/parsing/input/PagedSeq.scala (hunk 1 above)
Comment (julienrf) [reply shared/src/main/scala-2.11/scala/util/parsing/input/PagedSeq.scala]: Done!

**Comment by julienrf** [comment]
Yes, indeed. Sorry I forgot that one

**Pull request closed without merging**